tests/e2e/suite_test.go

```diff
@@ -54,7 +54,7 @@ func TestE2E(t *testing.T) {
 }
 
 var _ = BeforeSuite(func() {
-	k8sVersion = os.Getenv("INSTALL_K8S_VERSION")
+	k8sVersion = os.Getenv("INSTALL_K3S_VERSION")
```

Comment:
This is normal: variable `INSTALL_K3S_VERSION` is used by K3s installer. I will change this to something more modular when I will add RKE2 test.